src/models/Letter.model.ts

```diff
@@ -0,0 +1,19 @@
+export interface Letter {
+  letterType: LetterType;
+  paragraphID: ParagraphID;
+  letterIssuer: string;
+  letterDateRequested: string;
+}
+
+export enum LetterType {
+  TRL_AUTHORISATION = 'trailer authorisation',
+  TRL_REJECTION = 'trailer rejection',
```

Comment:
Could we use 'accept' and 'reject' instead of authorise please?

```suggestion
  TRL_ACCEPTANCE = 'trailer acceptance',
  TRL_REJECTION = 'trailer rejection',
```

Reply:
Is it not authorisation? That's what I've been seeing in the tickets